Tests/FoundationEssentialsTests/FileManager/FileManagerTests.swift

```diff
@@ -23,6 +23,10 @@ import TestSupport
 @testable import Foundation
 #endif
 
+#if canImport(Android)
+import Android
+#endif
```

Comment:
This is the compilation error I get without it:
```
Tests/FoundationEssentialsTests/FileManager/FileManagerTests.swift:622:15: error: cannot find 'getuid' in scope
620 |     func testFileAccessAtPath() throws {
621 |         #if !os(Windows)
622 |         guard getuid() != 0 else {
    |               `- error: cannot find 'getuid' in scope
```
There are some strange issues with these platform overlays, [which I investigated a bit last month](https://github.com/apple/swift-certificates/pull/183#discussion_r1730366251). Since nobody I asked seems to have a good idea of what's going on, I chalked it up to some Swift module leak that can be fixed later and am just adding these imports now to shut the compiler up.